src/modules/powerrename/dll/dllmain.cpp

```diff
@@ -203,38 +205,38 @@ class PowerRenameModule : public PowertoyModuleIface
 
         // Create a Settings object.
         PowerToysSettings::Settings settings(hinstance, get_name());
-        settings.set_description(L"A Windows Shell Extension for more advanced bulk renaming using search and replace or regular expressions.");
-        settings.set_icon_key(L"pt-power-rename");
+        settings.set_description(GET_RESOURCE_STRING(IDS_SETTINGS_DESCRIPTION));
+        settings.set_icon_key(GET_RESOURCE_STRING(IDS_SETTINGS_ICON));

```

Comment:
I think we should move those "id's" also from the resource file. Those would be:
```
IDS_SETTINGS_ICON        L"pt-power-rename" 
IDS_BOOL_PERSIST L"bool_persist_input" 
IDS_MRU_ENABLED L"bool_mru_enabled" 
IDS_MAX_MRU_SIZE L"int_max_mru_size" 
IDS_SHOW_ICON L"bool_show_icon_on_menu" 
IDS_EXTENDED_MENU L"bool_show_extended_menu"
```

While at it:
```
IDS_EXTENDED_MENU_INFO L"Only show the PowerRename menu item on the extended context menu (SHIFT + Right-click)." 
```
Can we make `SHIFT` into `Shift`?

Reply:
Sure! I shall make those changes.